### ISSUE TITLE
为什么loss出错

### ISSUE THREAD

**Issue by dajuzi123** [chenyuntc/pytorch-book#122]

**Comment by cheer00** [comment]
第二章

```
In[39] 
output = net(input)
target = t.arange(0,10).view(1,10) 
criterion = nn.MSELoss()
loss = criterion(output, target)
loss # loss是个scalar
```
运行到loss时报错： Expected object of type torch.FloatTensor but found type torch.LongTensor for argument #2 'target'


请问是什么原因

**Comment by dajuzi123** [comment]
把target = t.arange(0,10).view(1,10)这句改为:
target=t.arange(0,10).view(1,10).float()

**Comment by jokerwenxiao** [comment]
loss = criterion(output.float(), target.float())
